jaq-std/src/std.jq

```diff
@@ -22,6 +22,16 @@ def isinfinite: . == infinite or  . == -infinite;
 def isfinite:   isnumber and (isinfinite | not);
 def isnormal:   isnumber and ((. == 0 or isnan or isinfinite) | not);
 
+# Math
+def logb:
+    if . == 0 then -infinite
+  elif isinfinite then infinite
+  elif isnan then .
+  else ilogb_ end;
+def pow10:       pow(10; .);
+def significand: scalb(.; ilogb_ | -1 * .);
```

Comment:
Does this yield the same output as jq on nan, infinity, 0 and so on?
Some tests would be nice here.